README.md

```diff
@@ -532,6 +532,120 @@ or the `useSignTransactions` hook defined below. If you don't use one of these,
+
+It can be used to send a group of transactions (that ca be synchronized) with minimum information:
+
+
+```typescript
+const { sessionId, error } = await sendBatchTransactions({
+    transactions: [
+      [
+        {
+          value: '1000000000000000000',
+          data: 'tx1',
+          receiver: receiverAddress
+        },
+      ],
+      [
+        {
+          value: '1000000000000000000',
+          data: 'tx2',
+          receiver: receiverAddress
+        },
+        {
+          value: '1000000000000000000',
+          data: 'tx3',
+          receiver: receiverAddress
+        },
+      ]
+    ],
+  /**
+   * the route to be redirected to after signing. Will not redirect if the user is already on the specified route
+   * @default window.location.pathname
+   */
+  callbackRoute?: string;
+  /**
+   * custom message for toasts texts
+   * @default null
+   */
+  transactionsDisplayInfo: TransactionsDisplayInfoType;
+  /**
+   * Minimum amount of gas in order to process the transaction.
+   * @default 50_000
+   */
+  minGasLimit?: number (optional, defaults to 50_000);
+  /**
+   * Contains extra sessionInformation that will be passed back via getSignedTransactions hook
+   */
+  sessionInformation?: any (optional, defaults to null)
+  /**
+   * The transactions will be signed without being sent to the blockchain
+   */
+  signWithoutSending?: boolean;
+  /**
+   * Delay the transaction status from going into "successful" state
+   */
+  completedTransactionsDelay?: number;
+  /**
+   * If true, redirects to the provided callbackRoute
+   */
+  redirectAfterSigning?: boolean;
+  });
+```
+
+It returns a Promise that will be fulfilled with `{error?: string; sessionId: string | null;}`
+
+- `sessionId` is the transactions' session id used to compose the `batchId` which will be used to send the batch to the batch service and to track the transactions status and react to it.
+- `error` is the error that can appear during the signing/sending process.
+
+### How to synchronize transactions ?
+`sendBatchTransactions` accepts an argument `transactions` which is an array of transactions arrays.
+Each transaction array will be sent to the blockchain in the order they are provided.
+Having the example above, the transactions will be sent in the following order:
+- `tx1`
+- `tx2, tx3`
+
+`tx1` will be sent first, waits until is completed, then `tx2` and `tx3` will be sent in parallel. This means that the groups are sent synchronously, but the transactions inside a group are sent in parallel.
+
+** Important! This function will send the transactions automatically in batches, based on the provided transactions array, immediately after signing.
+If you do not want to be sent automatically, but on demand, then you should use send callback exposed by the `useSendBatchTransactions` hook.
+Be aware, you should take care of the `sessionId` passed to the batch. We recommend to generate a new sessionId like this: `Date.now().toString();` **
```

Comment:
Be aware, you should take care of the `sessionId` passed to the batch.
-->
Be sure to save the `sessionId` passed to the batch.